tools/docker-compose/ansible/roles/sources/templates/ldap.ldif.j2

```diff
@@ -84,3 +84,15 @@ objectClass: top
 objectClass: groupOfNames
 member: cn=awx_ldap_org_admin,ou=users,dc=example,dc=org
 
+{% if enable_ldap|bool and enable_vault|bool %}
+dn: cn=awx_ldap_vault,ou=users,dc=example,dc=org
+mail: admin@example.org
+sn: LdapAdmin
+cn: awx_ldap_admin
```

Comment:
use a different name here since awx_ldap_admin already exist

Reply:
something like hashicorp_ldap_admin?

Reply:
Will do. Probably a symptom of copy pasta here as I was walking out the door. Will change it tomorrow